OnlineHealthCenter/Services/Impressions/Impressions.API/Program.cs

```diff
@@ -0,0 +1,32 @@
+using Impression.API.Data;
+using Impression.API.Repositories;
+using Impression.API.Repositories.Interfaces;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddScoped<IImpressionContext, ImpressionContext>();
```

Comment:
Konfigurisanje servisa bih ja u nekom narednom refactor-u odradio u Common projektu - napravio bih ekstenziju nad service kolekcijom.
To svakako u okviru refaktora, ne sada.